### PR TITLE
CR 2: Zoom the text editor

### PR DESCRIPTION
**Pull request by evansalter**
Added menu items to zoom in/out of the text editor.

By "zoom", I assumed he just meant adjust the font size. Hopefully that assumption is correct...

### CODE REVIEW

**Comment by evansalter** [comment]
Writeup is here: https://docs.google.com/document/d/10FQwvwtq07D5rGByO8wjK_AAq10bTtTu-mdXvCwurRI/edit?usp=sharing

**Comment by evansalter** [comment]
I emailed Chanchal to make sure that by "zoom" he meant changing font size, and he said that's fine, so I think this is good to go.

**Review by ghost — APPROVED** [review]

**Review by coreyhickson — APPROVED** [review]

File: asn3/jEdit/org/gjt/sp/jedit/jedit_gui.props [hunk 1]
```diff
@@ -883,7 +888,7 @@ about.text=jEdit is brought to you by\n\
 	Ben Williams\n\
 	Bertalan Fodor\n\
 	Bill McMilleon\n\
-	Bj�rn "Vampire" Kautler\n\
+	Bj�rn "Vampire" Kautler\n\
```
Comment (coreyhickson) [inline asn3/jEdit/org/gjt/sp/jedit/jedit_gui.props]: Hey you got Bjorn to pop up in your PR as well, weird lol

Comment (ghost) [reply asn3/jEdit/org/gjt/sp/jedit/jedit_gui.props]: can't handle unicode?

Comment (evansalter) [reply asn3/jEdit/org/gjt/sp/jedit/jedit_gui.props]: Classic Eclipse!

**Pull request merged**